### PR TITLE
Remove bower

### PR DESCRIPTION
**Pull request by jasonk000**
Removes need for bower in the build process.

Simplifies dependency installation.
```
$ npm install
$ npm run build
$ npm run test
$ npm run serve
```
(Will require a documentation update for getvector.io)

### CODE REVIEW

File: gulp/conf.js [hunk 1]
```diff
@@ -39,3 +29,47 @@ exports.errorHandler = function(title) {
     this.emit('end');
   };
 };
+
+/**
+ * Most straightforward way to load dependencies in order with gulp
+ * Need to move to webpack
```
Comment (spiermar) [inline gulp/conf.js]: Assuming temporary until we migrate to webpack.

File: package.json [hunk 2]
```diff
@@ -6,9 +6,29 @@
...
+    "nvd3": "1.8.2"
+  },
   "scripts": {
-    "test": "gulp test"
+    "clean": "gulp clean",
+    "build": "gulp build",
+    "test": "gulp test",
+    "serve": "gulp serve"
   },
   "devDependencies": {
```
Comment (spiermar) [inline package.json]: Missing `editorconfig`.

File: gulp/conf.js (hunk 1 above)
Comment (jasonk000) [reply gulp/conf.js]: correct

File: package.json (hunk 2 above)
Comment (jasonk000) [reply package.json]: TIL editorconfig

Comment (jasonk000) [reply package.json]: @spiermar is `editorconfig` actually needed as a dep of this project? I don't see why editorconfig would need to be part of the project dev deps.

Comment (jasonk000) [reply package.json]: as discussed, will leave this out and do linter cleanups as a separate pr

**Pull request merged**